### ISSUE TITLE
Unable to use PowerShell 7.2.x in Azure Virtual Desktop

### ISSUE THREAD

**Issue by wdiggle** [PowerShell/PowerShell#17201]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

We are in the process of upgrading from PowerShell 7.0.2 to 7.2.2 and noticed that whilst via RDP you can type in to the command prompt when accessed via Azure Virtual Desktop all you can do is copy and paste in commands; you cannot type. Updated PSREADLINE module per another article reporting a similar issue with no success.

If I deploy the preview release of PowerShell 7.3.0 r3 the command prompt works exactly as it should with no issues both via an RDP session and in Azure Virtual Desktop.

### Expected behavior

```console
PS> Get-Module
```

### Actual behavior

```console
PS>
```

### Environment data

```powershell
PSVersion 7.2.3
PSEdition Core
GitCommitID 7.2.3
OS Microsoft Windows 10.0.19043
Platform Win32NT
PSCompatibleVersions {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion 1.1.0.1
WSManStackVersion 3.0
```

**Comment by daxian-dbw** [comment]
@SeeminglyScience Do you have access to an instance of Azure Virtual Desktop that can be used to reproduce this issue?

**Comment by SeeminglyScience** [comment]
I do not :/ PowerShell is blocked entirely on mine.

**Comment by theJasonHelmick** [comment]
Thank you @wdiggle  for submitting this issue. I haven't been able to repo this yet - our internal AVD is very restrictive.  Im in the process of accessing a full customer AVD to test this issue.

**Comment by theJasonHelmick** [comment]
@wdiggle - I spoke with the AVD team, and following their testing suggestions, could not reproduce this error.  My testing environment is not the same as the customers, and AVD recommends that customers experiencing this problem file an issue so they can attempt to repo.

The AVD team has recommended that customers file an issue in Feedback Hub from the AVD host encountering the issue.  Please select the category “Problem”, followed by “Apps” and “Remote Desktop”.  Screenshot below.

<img width="1158" alt="Screen Shot 2022-07-06 at 7 22 33 AM" src="https://user-images.githubusercontent.com/12662278/177576447-9c18eae1-58b8-4389-82bb-d0bbc782e595.png">
￼

**Comment by wdiggle** [comment]
The key to testing is using the web client and not the AVD remote desktop client which works fine. Knowing the AVD team they will have tested using the remote desktop client app. For the moment we are finding that the preview installations work fine so will continue with that for now.

Thanks.

------------------------------------------------------------
This email (including any attachments) may contain information that is confidential and legally privileged and which should not be disclosed. If you are not the intended recipient of this email, or you have received this email in error, any review, disclosure, copying, distribution or other use of (and/or acts or omissions in reliance on) its contents is strictly prohibited and you should notify the sender and delete the email (together with all copies and attachments) immediately.
We have used our reasonable endeavours to scan all attachments for viruses and other malicious or harmful code. However, we cannot accept liability for any loss or damage you may incur as a result of virus infection or other malicious or harmful code. You should ensure your systems protect against such threats.
Any personal information that you provide to us will be dealt with in line with our privacy policy, which explains what personal information we collect from you, how and why we collect, store, use and share such information, your rights in relation to your personal information and how to contact us and supervisory authorities in the event you have a query or complaint about the use of your personal information. Our privacy policy is available on request.

Telefónica Tech UK Limited is a limited company (Registered number: 2563193) registered in England and Wales whose registered office is at East House, New Pound Common, Wisborough Green, West Sussex, RH14 0AZ.
Telefónica Tech UK Managed Services Limited is a limited company (Registered number: 04434288) registered in England and Wales whose registered office is at East House, New Pound Common, Wisborough Green, West Sussex, RH14 0AZ.
Telefónica Tech Northern Ireland Limited is a limited company (Registered Number: NI028408) registered in Northern Ireland whose registered office is at Concourse Building 3, Queens Road, Belfast, Northern Ireland, BT3 9DT.
Telefónica Tech Ireland Limited is a limited company (Registered Number: 438959) registered in Ireland whose registered office is at Unit 5B Unit 5H, Fingal Bay Business Park, Balbriggan, Co.Dublin, Dublin, Ireland.

Telefónica Tech UK&I, previously known as CANCOM UK&I, was acquired by Telefónica Tech in July 2021. In the UK and Ireland, Telefónica Tech is the brand name for the following legal entities: Telefónica Tech UK Limited*, Telefónica Tech UK Managed Services Limited*, Telefónica Tech Northern Ireland Limited* and Telefónica Tech Ireland Limited*. Telefónica Tech will be used to refer to any or all of these entities throughout this document/communication. 

*All four were CANCOM entities until their official name change to Telefónica Tech at the start of October 2021. 
------------------------------------------------------------

**Comment by daxian-dbw** [comment]
@wdiggle This was already fixed in the 2.2 version of PSReadLine by https://github.com/PowerShell/PSReadLine/pull/2632.

The v7.2.x of PowerShell ships the v2.1.0 PSReadLine which doesn't have that fix. Please upgrade to [v2.2.6](https://www.powershellgallery.com/packages/PSReadLine/2.2.6) of PSReadLine. See https://github.com/PowerShell/psreadline#upgrading for instructions if you are not familiar with upgrading PSReadLine.

For details about how the root cause is discovered, see the comment thread starting at https://github.com/PowerShell/PowerShell/issues/17783#issuecomment-1211236702.
